akka-http/src/main/scala/akka/http/scaladsl/server/directives/FileAndResourceDirectives.scala

```diff
@@ -186,6 +181,11 @@ trait FileAndResourceDirectives {
 }
 
 object FileAndResourceDirectives extends FileAndResourceDirectives {
+  private val withRangeSupportAndPrecompressedMediaTypeSupportAndExtractSettings =
```

Comment:
Because with `wrap` I'd have to look at the source to see what exactly it's doing...